packages/application-components/src/components/maintenance-page-layout/maintenance-page-layout.spec.tsx

```diff
@@ -29,9 +29,9 @@ describe('rendering', () => {
     });
     it('renders the title and paragraphs', () => {
       const { getByText } = render(<MaintenancePageLayout {...props} />);
-      expect(getByText(props.title)).toBeInTheDocument();
-      expect(getByText(props.paragraph1)).toBeInTheDocument();
-      expect(getByText(props.paragraph2)).toBeInTheDocument();
+      expect(getByText('title')).toBeInTheDocument();
```

Comment:
why do you need to use the strings explicitly like this?

Reply:
Because those props are typed as react nodes but the query matcher only accepts a string or regex. We could type cast `as string` but we also usually explicitly pass the query text as string in our RTL tests.
So yeah, a bit unfortunate but it’s better like this.